### PR TITLE
For Prompt widget, Add __('<your_string>') method

### PR DESCRIPTION
**Pull request by hardikm-cirkleinc**
## Purpose of this pull request

For Prompt widget, Add __('<your_string>') method in Example because this method is used for translation of the string.

This pull request (PR) ...

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_prompt.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_prompt.html
- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_prompt.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/modal/prompt.js
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Ui/view/base/web/js/modal/prompt.js
- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Ui/view/base/web/js/modal/prompt.js

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/javascript-dev-guide/widgets/widget_prompt.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The prompt widget can be initialized with or without binding to a certain elemen
 
 ```javascript
 $('#prompt_init').prompt({
-    title: 'Prompt title',
+    title: jQuery.mage.__('Prompt title'),
```
Comment (osrecio) [inline guides/v2.1/javascript-dev-guide/widgets/widget_prompt.md]:
```suggestion
    title: $.mage.__('Prompt title'),
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_prompt.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ require([
 ], function(prompt) { // Variable that represents the `prompt` function
 
     prompt({
-        title: 'Some title',
-        content: 'Some content',
+        title: jQuery.mage.__('Some title'),
```
Comment (osrecio) [inline guides/v2.1/javascript-dev-guide/widgets/widget_prompt.md]:
```suggestion
        title: $.mage.__('Some title'),
```

File: guides/v2.1/javascript-dev-guide/widgets/widget_prompt.md [hunk 3]
```diff
@@ -39,8 +39,8 @@ require([
...
+        content: jQuery.mage.__('Some content'),
```
Comment (osrecio) [inline guides/v2.1/javascript-dev-guide/widgets/widget_prompt.md]:
```suggestion
        content: $.mage.__('Some content'),
```

**Comment by hardikm-cirkleinc** [comment]
Thanks @osrecio

**Pull request merged**